src/core_plugins/getting_started/public/components/getting_started/getting_started.html

```diff
@@ -0,0 +1,135 @@
+<div class="kuiViewContent gettingStartedContent">
+
+  <div
+    ng-if="!gettingStarted.hasOptedOut"
+    class="kuiBar gettingStartedOptOutContent"
+  >
+    <div class="kuiBarSection">
+      <p class="kuiText kuiSubduedText">
+        <a
+          kbn-href="#/management"
+          ng-click="gettingStarted.recordOptOut()"
+        >I'm a pro, skip this stuff &gt;</a>
```

Comment:
On further thought, this link seems really deceptive and non-helpful. If you're a new user, you're not going to want to click it because you're "not a pro". But in this case, what do you do? There's no clear next step into Kibana.

I think we should remove this link and instead add a big fat call to action button below the cards, that says something to the effect of, "OK I'm ready to do great things!" with some small subtext below it that says "You can see this page at any time from within the Management app."

This way both new users and "pro" users have a clear exit point, and reassurance that they're not going to lose access to all of these helpful links and resources.